### DISCUSSION TITLE
Dependabot Does Not See Upgrades in Build kts Files When Using TOML Catalog

### DISCUSSION THREAD

**Discussion by grndvl1** [community/community#38024] [Code Security]
Everyone of my dependencies is in a tool file and using the Dependencies.  Dependabot does not see this and cannot show me upgrades.  Using Kotlin Gradle scripts with catalogue file in gradle directory toml file

**Comment by ossanna16** [comment]
Hi there @grndvl1 and welcome to our community! Thank you for asking a great question 🙂 Can you please provide screenshots so we can better understand your issue?
